### PR TITLE
Cleanup dependencies and transpile command

### PR DESCRIPTION
**Pull request by adamjarling**
This PR adds in a `clean` command, to ensure that when packaging the files we start with a clean directory each time.

Also Readme updated.

### CODE REVIEW

**Comment by Dananji** [comment]
@adamjarling All looks great! :+1:

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # React Structural Metadata Editor
 
-A React component which displays structural metadata about an ingested work, and a visualized waveform to help navigating sections of the audio waveform. A user can add, edit and delete headers and timespans within the structural metadata, and save the record into a consuming application.
+A React component which displays structural metadata about an ingested audio/video file, and displays a visualized waveform to help navigating sections of the audio waveform. A user can add, edit and delete headers and timespans within the structural metadata, and save the record into a consuming application.
+
+**Note**: We are not currently publishing this package to NPM, but rather consuming it directly via a Github repository URL address. See **Deployment** notes below for more info and example code.
```
Comment (Dananji) [inline README.md]: @adamjarling Do you think it's better if we link the `Deployment` section here?

File: README.md [hunk 2]
```diff
@@ -35,17 +37,17 @@ yarn install
...
 
 ### Development
 
 All development should be done in the `/src` directory.
 
-The `/demo` directory contains demo application entry point `index.html` file, and mocks Avalon consuming the SME component.
+The `/demo` directory contains the demo application entry point `index.html` file, and mocks Avalon consuming the SME component.
 
-If you'd like to try out a new feature for SME without merging into `master`, create a new branch, ie: `my-new-feature`, and push it up to `https://github.com/avalonmediasystem/react-structural-metadata-editor` See the Deployment section below on how to import from either `master` or your own feature branch.
+If you'd like to try out a new development feature before merging your code into `master`, create a new branch, ie: `my-new-feature`, and push your branch to `https://github.com/avalonmediasystem/react-structural-metadata-editor`. See the Deployment section below on how to import from either `master` or your own feature branch.
```
Comment (Dananji) [inline README.md]: And here as well?

File: README.md (hunk 1 above)
Comment (adamjarling) [reply README.md]: Yes, good idea.

**Pull request merged**